Greenstand-Token.md

```diff
@@ -551,6 +544,10 @@ A unique token with unique attributes.
 
 A Root Wallet is a level of wallet permission on a Greenstand node that enables a user to make API calls that trigger the creation of new wallets and transfer requests from their own or managed wallets to any other wallet in the system. Root Wallets may grant wallet trading access to specific wallets for other users, third parties, and customers.
 
+**Seed**
+
+A Seed is 100th of a TRing.
+
```

Comment:
https://academy.binance.com/en/glossary/wei
In my opinion, we should conform to the existing standard to the best of our ability. I will defer to your judgement, however.